app/Resources/views/articleInfo/result.html.twig

```diff
@@ -260,22 +277,22 @@
                         <td>{{ msg('links-external') }}</td>
                         <td>{{ ai.linksExtCount|num_format }}</td>
                     </tr>
-                    {% if page.watchers is not empty %}
+                    {% if page.watchers is not empty and not (ai.hasDateRange) %}
```

Comment:
`not (ai.hasDateRange)` first. Here I think `Page::getPageInfo()` has already be ran, but good to be consistent with the placement of the hasDateRange check.